src/util/loanUtils.js

```diff
@@ -270,7 +271,7 @@ export function watchLoanCardData({
 }
 
 export function readLoanFragment({
-	apollo, loanId, fragment
+	apollo, loanId, fragment, publicId
```

Comment:
Does adding this new variable cause any issues with the other usages of these utils? As in, is loan data still successfully pulled from the cache and watched?

Reply:
This is also used in `KvClassicLoanCardContainer` that I tested and working fine.